Tests/MongoSwiftTests/EventLoopBoundMongoClientTests.swift

```diff
@@ -320,7 +320,7 @@ final class EventLoopBoundMongoClientTests: MongoSwiftTestCase {
             defer { try? db.drop().wait() }
 
             let wrongEventLoop = MultiThreadedEventLoopGroup(numberOfThreads: 4).next()
-            let withSessionFuture: EventLoopFuture<[String]> = elBoundClient.withSession { session in
+            let withSessionFuture = elBoundClient.withSession { session in
```

Comment:
the explicit annotation is not needed anymore, and that saves us importing NIOCore here